Tests/PassesTests/PassesTests.swift

```diff
@@ -1,15 +1,270 @@
 import XCTVapor
+import Fluent
+import FluentSQLiteDriver
 @testable import Passes
+import PassKit
 
 final class PassesTests: XCTestCase {
-    func testExample() {
-        // This is an example of a functional test case.
-        // Use XCTAssert and related functions to verify your tests produce the correct
-        // results.
-        //XCTAssertEqual(PassesService().text, "Hello, World!")
+    let passDelegate = TestPassesDelegate()
+    let passesURI = "/api/passes/v1/"
+    var passesService: PassesService!
+    var app: Application!
+    
+    override func setUp() async throws {
+        self.app = try await Application.make(.testing)
+        app.databases.use(.sqlite(.memory), as: .sqlite)
+        
+        PassesService.register(migrations: app.migrations)
+        app.migrations.add(CreatePassData())
+        passesService = try PassesService(
+            app: app,
+            delegate: passDelegate,
+            pushRoutesMiddleware: SecretMiddleware(secret: "foo"),
+            logger: app.logger
+        )
+        app.databases.middleware.use(PassDataMiddleware(service: passesService), on: .sqlite)
+
+        try await app.autoMigrate()
+    }
+
+    override func tearDown() async throws { 
+        try await app.autoRevert()
+        try await self.app.asyncShutdown()
+        self.app = nil
+    }
+
+    func testPassGeneration() async throws {
+        let passData = PassData(title: "Test Pass")
+        try await passData.create(on: app.db)
+        let pass = try await passData.$pass.get(on: app.db)
+        let data = try await passesService.generatePassContent(for: pass, on: app.db)
+        XCTAssertNotNil(data)
```

Comment:
Is there a better way to test the generated passes? Testing from some `Data` is a little brittle